hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -597,6 +597,18 @@
       ozone.om.http-address.
     </description>
   </property>
+  <property>
+    <name>ozone.om.http.policy</name>
+    <value>HTTP_ONLY</value>
+    <tag>OM, MANAGEMENT</tag>
```

Comment:
Just want to tag this:
As OzoneManagerHttpServer uses BaseHttpServer, and that uses dfs.http.policy to find when it should use http/httpsAddress.

With this newly added property, we should pass this policy when OzoneManagerHtttpServer is started, to updateConnectorAddress during httpServer Start.

Reply:
Updated BaseHttpServer to take ozone.http.policy if set. Otherwise, use dfs.http.policy.